### ISSUE TITLE
On newset pocketbook firmware, screen is rotated, but touch not

### ISSUE THREAD

**Issue by mstrychalski** [koreader/koreader#7747]
Koreader works fine, but when opening book, splash screen is rotated, then text itself too, but touch is in regular rotation. It is unusable, even when somehow enabling 90 degree rotation, it goes to default position, but now touch events are rotated 90 degrees. You can't touch what you see on screen that way.

Device is PocketBook LUX 4

**Comment by mergen3107** [comment]
What orientation of the device do you start from? 
There were similar issues when KOReader was launched from "upside down" or landscape orientation.

**Comment by mstrychalski** [comment]
Its default portrait orientation, I didn't changed anything. I updated firware today, I've got system message from auto updater, after that, koreader was like that. I deleted it from applications and system folders, downloaded latest release, but still rotated. Display is rotated and in koreader settings, it shows 0 degrees, but screen is rotated, and touch events not.

**Comment by NiLuJe** [comment]
You've completely ignored the issue template, which means I have no idea which KOReader version you're running.

This might have been fixed recently, or it might be one of the myriad pocketbook related quirks with newer FW (I can't for the life of me find the relevant issue number right now).

**Comment by mstrychalski** [comment]
I downloaded latest Koreader version - 2021.05 "Yellow Daffodil", I've mentioned it in my reply. I am using latest firware version of pocketbook lux 4. Its U627.6.3.569

**Comment by mstrychalski** [comment]
* KOReader version: 2021.05 "Yellow Daffodil"
* Device: pocketbook lux 4 (fw ver. U627.6.3.569)

#### Issue
Koreader works fine, but when opening book, splash screen is rotated, then text itself too, but touch is in regular rotation. It is unusable, even when somehow enabling 90 degree rotation, it goes to default position, but now touch events are rotated 90 degrees. You can't touch what you see on screen that way.

Device is PocketBook LUX 4

#### Steps to reproduce
Open any book

##### `crash.log` (if applicable)
Not applicable

I did that now :D

**Comment by NiLuJe** [comment]
Well, that's in direct contradiction to https://github.com/koreader/koreader/issues/7663#issuecomment-841611429 (and the fix should have landed in 2021.05).

:s.

**Comment by LucyOne** [comment]
> 
> 
> Well, that's in direct contradiction to [#7663 (comment)](https://github.com/koreader/koreader/issues/7663#issuecomment-841611429) (and the fix should have landed in 2021.05).

Has device.lua been updated to include `isAlwaysPortrait = yes,` for the Touch Lux 4? This was what the user from the German forum for whom I was reporting had done. 

Slightly related, he also fixed KOReader support for the InkPad Color, by inserting:
```
-- PocketBook InkPad Color (741)
local PocketBook741 = PocketBook:new{
    model = "PBInkPadColor",
    display_dpi = 300,
    hasColorScreen = yes,
    canUseCBB = no, -- 24bpp
    isAlwaysPortrait = yes,
    usingForcedRotation = landscape_ccw,
}
```

and

`elseif codename == "PB741" then
    return PocketBook741`

**Comment by Jakski** [comment]
I experienced the same problem as @mstrychalski after upgrading PocketBoot firmware. Reinstalling KOReader yielded no results. I added `isAlwaysPortrait = yes,` to https://github.com/koreader/koreader/blob/b938f312a6245f25a2e9713c7da4787603973c43/frontend/device/pocketbook/device.lua#L488 and KOReader became usable again. Thank you for sharing solution @LucyOne.

My device:
````
Device: PocketBook Lux 4
Model: PB627
Firmware: U627.6.3.569
KOReader: 2021.05 "Yellow Daffodil"
````
